handler.py

```diff
@@ -125,18 +125,17 @@ def _process_directory(repo, branch_name, base_server_path, server_path, version
                     key = None
 
                     if relative_path in UNVERSIONED_FILES:
-                        expanded_file_data = json_data
                         key = relative_path
                     else:
-                        schema_url = SCHEMA_URL.get(branch_name)
-                        release_preparation = ReleasePreparation(schema_url=schema_url, version_map=version_numbers)
-                        expanded_file_data = release_preparation.expand_urls(relative_path, json_data)
-                        key = release_preparation.get_schema_key(expanded_file_data)
+                        schema_base_url = SCHEMA_URL.get(branch_name)
+                        metadata_schema = MetadataSchema(json_data, relative_path)
+                        json_data = metadata_schema.get_json_schema(version_numbers, schema_base_url)
+                        key = get_relative_url(relative_path, version_numbers)
 
-                    if key is None:
+                    if not key:
```

Comment:
Good catch. I think i have thought of checking for empty strings as well but I realised it might be also a valid value. I've reverted to the `key is None` condition. Thanks!